back/src/adapters/secondary/pg/repositories/PgFormEstablishmentRepository.ts

```diff
@@ -10,130 +16,140 @@ import {
   ConflictError,
   NotFoundError,
 } from "../../../primary/helpers/httpErrors";
-import { executeKyselyRawSqlQuery, KyselyDb } from "../kysely/kyselyUtils";
-import { optional } from "../pgUtils";
+import {
+  jsonBuildObject,
+  jsonStripNulls,
+  KyselyDb,
+} from "../kysely/kyselyUtils";
 
 const logger = createLogger(__filename);
 export class PgFormEstablishmentRepository
   implements FormEstablishmentRepository
 {
   constructor(private transaction: KyselyDb) {}
 
-  public async create(
-    formEstablishmentDto: FormEstablishmentDto,
-  ): Promise<void> {
-    // prettier-ignore
-    const {  siret, source, businessName, businessNameCustomized, businessAddress, isEngagedEnterprise, naf, appellations: professions, businessContact,  website, additionalInformation, fitForDisabledWorkers, maxContactsPerWeek } =
-      formEstablishmentDto
-
-    const query = `INSERT INTO form_establishments(
-        siret, source, business_name, business_name_customized, business_address, website, additional_information, is_engaged_enterprise, naf, professions, business_contact, fit_for_disabled_workers, max_contacts_per_week
-      ) VALUES($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13)`;
-
-    // prettier-ignore
-    try {
-      await executeKyselyRawSqlQuery(
-        this.transaction,
-        query,
-        [siret, source, businessName, businessNameCustomized, businessAddress, website, additionalInformation, isEngagedEnterprise, naf, JSON.stringify(professions), JSON.stringify(businessContact), fitForDisabledWorkers, maxContactsPerWeek]
-      );
-    } catch (error: any) {
-      logger.error({error}, "Cannot save form establishment ")
-      notifyObjectDiscord({
-        _message: `Cannot create form establishment with siret ${formEstablishmentDto.siret}`,
-        ...error
+  public async create(formEstablishment: FormEstablishmentDto): Promise<void> {
+    return this.transaction
+      .insertInto("form_establishments")
+      .values({
+        additional_information: formEstablishment.additionalInformation ?? null,
```

Comment:
est-ce que ça ne suffirait:
```suggestion
        additional_information: formEstablishment.additionalInformation,
```
?